docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ImageAnalytics/ExtractPixels.cs

```diff
@@ -38,24 +40,49 @@ public static void Example()
             var imagesFolder = Path.GetDirectoryName(imagesDataFile);
             // Image loading pipeline. 
             var pipeline = mlContext.Transforms.LoadImages("ImageObject", imagesFolder, "ImagePath")
-                          .Append(mlContext.Transforms.ResizeImages("ImageObject", imageWidth: 100, imageHeight: 100 ))
-                          .Append(mlContext.Transforms.ExtractPixels("Pixels", "ImageObject"));
+                          .Append(mlContext.Transforms.ResizeImages("ImageObjectResized", inputColumnName:"ImageObject", imageWidth: 100, imageHeight: 100 ))
+                          .Append(mlContext.Transforms.ExtractPixels("Pixels", "ImageObjectResized"));
 
 
             var transformedData = pipeline.Fit(data).Transform(data);
 
             // The transformedData IDataView contains the loaded images now
-            //Preview of the transformedData
-            var transformedDataPreview = transformedData.Preview();
 
-            // Preview of the content of the images.tsv file
-            //
-            // ImagePath    Name        ImageObject                 "Pixels"
```

Comment:
> // ImagePath    Name        ImageObject  [](start = 11, length = 41)

maybe keep the headers/column names, even if they don't print out. Or print them separately.

Reply:
fixed

---
In reply to: [271136252](https://github.com/dotnet/machinelearning/pull/3165#discussion_r271136252) [](ancestors = 271136252)